README.md

```diff
@@ -73,7 +73,12 @@ terraform destroy
 
 ### ERD diagram
 
-![ERD diagram](...)
+![ERD diagram](https://drawsql.app/teams/isaac-11/diagrams/railway-tracker-3)
+
+![image](https://github.com/adamski201/Railway-Tracker/assets/84942651/9e6dc207-77bf-41b4-b83a-ae77c5825deb)
```

Comment:
Looks really good!! very clear and consistent styling

Reply:
Good idea - I've made those changes and pushed, so should be fixed now!